OpenRA.Utility/Program.cs

```diff
@@ -28,7 +28,6 @@ static void Main(string[] args)
 			AppDomain.CurrentDomain.AssemblyResolve += GlobalFileSystem.ResolveAssembly;
 
 			Log.AddChannel("perf", null);
-			Log.AddChannel("debug", null);
```

Comment:
No, sorry. The reason I removed that is once a channel is added, the subsequent calls to add the same channel are ignored. So once it is added as null, it is not possible to change that channel to debug.log.